.github/workflows/release.yml

```diff
@@ -0,0 +1,66 @@
+on:
+  push:
+    tags:
+      - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+name: Release workflow
+jobs:
+  build-phar:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@master
+        with:
+          php-version: 7.4
+          ini-values: memory_limit=2G, display_errors=On, error_reporting=-1, phar.readonly=0
+          tools: pecl, phive
+
+      - name: Get composer cache directory
+        id: composer-cache
+        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+      - name: Cache composer dependencies
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: composer-${{ hashFiles('**/composer.lock') }}
+          restore-keys: composer-
+
+      - name: Install Composer dependencies
+        run: |
+          composer install --no-progress --prefer-dist --optimize-autoloader
+
+      - name: Install phive tools
+        run: phive install --trust-gpg-keys 2A8299CE842DD38C
+
+      - name: Create build dir
+        run: make phar
+
+      - name: Upload PHAR file
+        uses: actions/upload-artifact@v2.2.0
+        with:
+          name: prophecy-phpunit.phar
+          path: build/prophecy-phpunit.phar
+
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
should use `github.token` to use the default token available in github actions. AFAICT, there is no need for a token with wider permissions here